sass/_page.sass

```diff
@@ -237,6 +237,93 @@ button
         &.underline-from-right
             @include underline(right, $activeColor, 5px)
 
+
+        // Icon Transitions
+        &.icon-back
+            @include icon-back
```

Comment:
that doesn't work for mixin names, I'm all about the loops so I definitely tried it